besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2120,8 +2120,6 @@ private void configure() throws Exception {
     instantiateSignatureAlgorithmFactory();
 
     logger.info(generateConfigurationOverview());
-    logger.info("Connecting to {} static nodes.", staticNodes.size());
-    logger.trace("Static Nodes = {}", staticNodes);
```

Comment:
Note moving the log from here to where the configuration is happening highlights that the updateNetworkConfig() method is called twice. Not sure if that is intentional